app/controllers/pets_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def show
     @pet = Pet.find(params[:id])
+    @favorites= Favorites.new(session[:favorites])
```

Comment:
Will we need to add this to every action that has a corresponding view?

Reply:
I think so

Reply:
I think so